src/idsk/components/footer-extended/footer-extended.scss

```diff
@@ -96,80 +81,313 @@
+            flex-wrap: wrap; // Support: Flexbox
+        -webkit-box-align: start;
+            -ms-flex-align: start;
+                align-items: flex-start; // Support: Flexbox
+        -webkit-box-pack: center;
+            -ms-flex-pack: center;
+                justify-content: center; // Support: Flexbox
+      }
+
+    .idsk-footer-extended-meta-item {
         margin-right: 15px;
         margin-bottom: 25px;
-        margin-left: 0px;
+        margin-left: 0;
         padding-bottom: 20px;
     }
 
-    .logo {
+    .idsk-footer-extended-meta-item--grow {
+        flex: 1; // Support: Flexbox
+        @include mq ($until: tablet) {
+          flex-basis: 320px; // Support: Flexbox
+        }
+      }
+
+    .idsk-ministri-logo {
         flex: none;
         order: 1;
         align-self: flex-end;
         flex-grow: 0;
         padding-top: 8px;
     }
 
-    .frame {
+    .idsk-frame {
         display: inline-block;
         width: 100%;
         height: 30px;
-        left: 0px;
-        top: 0px;
-
+        left: 0;
+        top: 0;
         font-style: normal;
         font-weight: 400;
         font-size: 17px;
         line-height: 25px;
-
-        color: #0b0c0c;
+        color: govuk-colour("black");
     }
 
-    .label {
+    .idsk-label {
         font-style: normal;
         font-weight: normal;
         font-size: 19px;
         line-height: 25px;
-
-        color: #0b0c0c;
+        color: govuk-colour("black");
         text-decoration: underline;
     }
 
-    .bottom-space {
+    .idsk-grid-column {
         padding-bottom: 23px;
     }
 
+    .idsk-text-h3 {
+        font-family: Source Sans Pro;
+        font-style: normal;
+        font-weight: normal;
+        font-size: 21px;
+        line-height: 25px;
+        color: govuk-colour("black");
+        font-weight: bold;
+    }
+
+    .heading-feedback {
+        margin: 0 -15px;
+        background: govuk-colour("blue");
+    }
+
+    .feedback-text {
+        font-family: Source Sans Pro;
+        font-style: normal;
+        font-weight: bold;
+        font-size: 12px;
+        line-height: 23px;
+        color: govuk-colour("white");
+    }
+
+    @include mq(tablet) {
+        .govuk-heading-m {
+            font-size: 1.39em;
+        }
+    }
+
+    @include mq($from: tablet) {
+        .feedback-text {
+            top: 7.49%;
+            bottom: 91.98%;
+            font-family: Source Sans Pro;
+            font-style: normal;
+            font-weight: bold;
+            font-size: 18px;
+            line-height: 23px;
+            color: govuk-colour("white");
+        }
+    }
+
+    .right-text {
+        float: right;
+    }
+
+    @include mq(tablet) {
+        .text-padding {
+            padding-left: 34px;
+        }
+    }
+
+    @include mq($from: desktop) {
+        .text-padding {
+            padding-left: 70px;
+        }
+
+        .govuk-heading-m {
+            font-size: 1.5em;
+        }
+    }
+    .feedback-container {
```

Comment:
tu by som tiez dal idsk-feedback-container ak je to tebou vytvoreny container nez len feedback container..a to iste nizsie